std_msgs/Header header

uint8 event_id
uint32 time_stamp
uint32 measurement_counter
float32 vambig
float32 center_frequency
RadarDetection[] detections